apps/electron-app/scripts/build-dock.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Note: Do not run this script directly, run `npm run build` in the `demo` folder to make the variables available.
+
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)